crates/bevy_pbr/src/deferred/pbr_deferred_functions.wgsl

```diff
@@ -80,9 +80,11 @@ fn pbr_input_from_deferred_gbuffer(frag_coord: vec4<f32>, gbuffer: vec4<u32>) ->
     let props = deferred_types::unpack_unorm3x4_plus_unorm_20_(gbuffer.b);
     // Bias to 0.5 since that's the value for almost all materials.
     pbr.material.reflectance = saturate(props.r - 0.03333333333);
+    pbr.specular_occlusion = props.b; // use diffuse occlusion as specular occlusion because we can't fit both in the g-buffer on webgl
```

Comment:
```suggestion
    pbr.specular_occlusion = props.b; // Treat diffuse occlusion as specular occlusion because we can't fit both in the g-buffer on webgl
```